docker-hub/scim.md

```diff
@@ -1,67 +1,151 @@
+You must run a full-sync after enabling SCIM, if you already have users assigned to the Docker Hub app. This provisions the users that are assigned in the IdP Directory to Docker Hub.
 
 1. Navigate to **Applications** > **Applications** and select the Docker Hub app.
 2. In the **Assignments** tab, click **Provision User** if you have pending users.
 3. Click **Apply to All** > **Reapply Mappings** and **Confirm**.
 
     > **Note**
     >
-    > Any user that was not previously provisioned is now provisioned in Docker Hub.
+    > This will generate a full-sync and any user that was not previously provisioned is now provisioned in Docker Hub.
+
+![provision-user](images/provision-user.png){:width="700px"}
+
+### Azure AD
+
+1. Navigate to Azure AD and select **AzureAD admin**.
+2. In the **Default Directory**, select **Add** > **Enterprise Application** > **Create your own application**.
+
+    > **Note**
+    >
+    > When you create an Enterprise Application you can configure SCIM with
+    > Security Assertion Markup Language (SAML).
+
+3. Type **Docker** for application name, select **non-gallery**, and click **Create**.
+4. In your Docker application, navigate to **Provisioning**, and click **Get Started**.
+5. Select **Automatic** for the provisioning mode and enter your SCIM credentials.
+
+    > **Note**
+    >
+    > You can access and copy your SCIM URL (Tenant URL) and API Token (Secret
+    > Token) in Docker Hub.
+
+6. Click **Test Connection** to enable and authorize the provisioning and click **Save**.
+7. In **Mappings**, click **Provision Azure Active Directory Groups** to disable Groups and click **Save**.
+8. Click **Provision Azure Active Directory Users**, in **Attribute Mappings**, and keep the **userName**, **Active**, **givenName**, **familyName**. Delete the other attributes listed.
+
+    ![attribute-mapping](images/attribute-mapping.png){:width="700px"}
+
+9. Click **Start Provisioning** to begin the full synchronization.
+
+     > **Note**
+     >
+     > It can take up to 30 minutes to begin provisioning. You can also
+     > provision on demand provisioning with one user that is already assigned
+     > to the application.
+
+### Onelogin
+
+1. In Onelogin, navigate to **Applications** > **Applications** > **Add app**.
+2. In the search field, enter **SCIM Provisioner with SAML (SCIM vs Core)** and select the item in the results.
+3. Enter **Docker Hub** as the display name and click **Save**.
+4. Navigate to the left navigation, and select **Configuration**.
+5. In a separate tab, navigate to **Docker Hub** > **Settings** > **Security** > **SSO** > **SAML**.
+6. Copy the following fields from Docker Hub, in to OneLogin:
+
+    * Entity ID: SAML Audience URL
+    * ACS URL:  SAML Consumer URL
+    * SCIM Base URL: SCIM Base URL
+    * Custom Headers: 'Content-Type: application/scim+json'
```

Comment:
suggestion: add a code snippert for the Custom Headers, so they can copy the values
```
Content-Type: application/scim+json
User-Agent: OneLogin SCIM
```

Reply:
Updated